src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs

```diff
@@ -23,17 +23,17 @@ public sealed class Cluster : IDeepCloneable<Cluster>
         /// <summary>
         /// Circuit breaker options.
         /// </summary>
-        public CircuitBreakerOptions CircuitBreakerOptions { get; set; }
+        public CircuitBreakerOptions CircuitBreaker { get; set; }
```

Comment:
Do some of these sound odd in the singular?
```suggestion
        public CircuitBreakerOptions CircuitBreakers { get; set; }
```
Also HealthChecks, Quotas. The others seem ok.

Reply:
So, should I change it to CircuitBreakers, HealthChecks, and Quotas?

Reply:
Yes, unless you can give a reason not to?

Reply:
I would prefer to leave it as is.

CircuitBreaker - [Circuit breaker design pattern](https://en.wikipedia.org/wiki/Circuit_breaker_design_pattern), but not Circuit breakers. 
HealthCheck - we set up options of one health check (timeout, interval etc.). There is no possibility to set up many checks, so, HealthCheck seems to be ok.
Quotas - don't know, can be changed

Reply:
Fair enough. We'll do an end-to-end API review later in the release.